packages/angular_devkit/build_angular/src/extract-i18n/ivy-extract-loader.ts

```diff
@@ -37,20 +34,22 @@ export default function localizeExtractLoader(
       console.debug(...args);
     },
     info(...args: string[]): void {
-      loaderContext.emitWarning(args.join(''));
+      loaderContext.emitWarning(new Error(args.join('')));
     },
     warn(...args: string[]): void {
-      loaderContext.emitWarning(args.join(''));
+      loaderContext.emitWarning(new Error(args.join('')));
     },
     error(...args: string[]): void {
-      loaderContext.emitError(args.join(''));
+      loaderContext.emitError(new Error(args.join('')));
     },
   };
 
   let filename = loaderContext.resourcePath;
-  if (map?.file) {
+  const mapObject =
```

Comment:
Looks like this like is causing the issue.

Reply:
Was the call to `this.getOptions`.  Accidentally left `this.getOptions(this)` which caused Webpack to treat `this` as a JSON schema.